src/systemTest/java/com/flightstats/hub/system/service/ChannelService.java

```diff
@@ -37,15 +47,19 @@ public String getChannelUrl(String channelName) {
         return getHubBaseUrl() + "channel/" + channelName;
     }
 
-    public HttpUrl getHubBaseUrl() {
+    private HttpUrl getHubBaseUrl() {
         return hubBaseUrl;
     }
 
     @SneakyThrows
     public void create(String channelName) {
         log.info("Create channel name {} ", channelName);
+        createCustom(Channel.builder().name(channelName).build());
+    }
 
-        Call<Object> call = channelResourceClient.create(Channel.builder().name(channelName).build());
+    @SneakyThrows
+    public void createCustom(Channel channel) {
```

Comment:
This is a trivial suggestion, but what do you thing about renaming `create` to `createWithDefaults` and then making `createCustom` just plain `create`?

Reply:
good point